data/strings/strings.txt

```diff
@@ -1930,6 +1934,7 @@
     comment = Free space out of total storage size in Maps Storage settings, e.g. "300 MB free of 2 GB"
     tags = android
     en = %1$@ free of %2$@
+    ar = %1$@ فارغ من %2$@
```

Comment:
Here 2% should be a continuous sequence, like `%1$@`: `%2$@`, otherwise, the string won't be formatted properly.

Reply:
The problem here is that Arabic is is right to left which causes many issues when trying to write English and Arabic together. I will fix it now. Thanks for your note

Reply:
After double-checking I see that despite it being displayed wrongly, actually it is stored in the right way. Can you please then undo the deletion? We will test how it work in the app.